workspaces/src/rpc/client.rs

```diff
@@ -208,7 +208,7 @@ impl Client {
             signer,
             receiver_id,
             TransferAction {
-                deposit: amount_yocto.as_yoctonear(),
+                deposit: NearToken::from_yoctonear(amount_yocto.as_yoctonear()),
```

Comment:
I believe that the `amount_yocto` is not an accurate variable name anymore as it seems to be a NearToken type and as such you don't need to convert it to yocto and back to NearToken at all.